airbyte-integrations/connectors/source-shopify/source_shopify/utils.py

```diff
@@ -88,38 +90,38 @@ def __new__(self, stream: str) -> Mapping[str, Any]:
         }
 
 
-class ShopifyAccessScopesError(Exception):
+class ShopifyAccessScopesError(AirbyteTracedException):
```

Comment:
For documentation purposes, the exceptions were updated following this [Sentry issue](https://airbytehq.sentry.io/issues/4373798948). Thanks Baz!